### ISSUE TITLE
WPE-2.38 : Navigation UI incorrect for Channel 4 app

### ISSUE THREAD

**Issue by suresh-khurdiya-epam** [Metrological/WebKitForWayland#1248]
Reproducton scenario:
[0] UK vpn needed
1. Run Channel 4 app 
2. Start to play video (some video are unable to play because of geolocation even with VPN)
3. While video playback press "Pause" RC button

Actual result:
UI incorrect - see attached video

Expected result:
UI correct

https://github.com/WebPlatformForEmbedded/WPEWebKit/assets/136487900/af461bc2-7a9f-455a-b5ed-7806ee2b462b

Sometime, we are seeing a glitch on video and some time glitch on image. In attached video,we can see glitch on Video.
So, basically, when we press Pause key then apps shows the image and pressing Play key it will remove and play the content.

channel 4 app url : https://virginall4.channel4.com/vmwebapp/index.html
- to reproduce need to:
    - change user agent to 
"Mozilla/5.0 (Linux armv7l) AppleWebKit/602.1.28+ (KHTML, like Gecko) Version/9.1 Safari/601.5.17 virginmedia Arris HZN/5.07"

  - test content on device which supports PlayReady DRM.

Note: We can register to app (registration is for free)

**Comment by magomez** [comment]
This could be related to https://github.com/WebPlatformForEmbedded/WPEWebKit/issues/1186. @suresh-khurdiya-epam could you update wpe-2.38 so it includes ee006a258bb754ca841b429c886c701ad1a5fe4f ? Maybe it fixes this as well.

**Comment by suresh-khurdiya-epam** [comment]
Thanks, Yes, with this PR #1186, my an issue also fixed.so we can close this request,
